receiver/oracledbreceiver/scraper_test.go

```diff
@@ -390,6 +390,34 @@ func TestSamplesQuery(t *testing.T) {
 	}
 }
 
+func TestGetInstanceId(t *testing.T) {
```

Comment:
Can you add a couple tests here for invalid instance IDs that fail to be parsed? I want to ensure we're getting back the default values as expected.